auth/README.md

```diff
@@ -0,0 +1,118 @@
+# Auth - Authentication and Authorization service
+
+Auth service provides authentication features as an API for managing authentication keys. User service is using AuthN service gRPC API to obtain login token or password reset token. Authentication key consists of the following fields:
```

Comment:
`...provides authentication and authorization... Auth service gRPC API...`
I would split this into 3 sections: Authentication (where we talk about Keys), Groups, and Authorization (where we talk about policies).

Reply:
yes, I would leave that for later, when everything is done